ledger/src/blockstore_processor.rs

```diff
@@ -1574,6 +1582,26 @@ pub fn fill_blockstore_slot_with_ticks(
     last_entry_hash
 }
 
+/// Check the transaction execution results to see if any instruction errored by exceeding the
+/// accounts data budget.  If yes, the whole block needs to be failed.
```

Comment:
nit: `accounts data budget` -> `max accounts size limit for (rooted slots? all slots?)`

Reply:
Yah, it's all slots. It's really trying to capture the total size of AccountsDb.

Reply:
Hmmm how does this remain consistent across nodes who see some forks, and others see other forks?

For instance one validator replaying slot 3 sees

```
0 - 1 - 3
```

the other sees

```
      | -2
0 - 1 - 3
```

I'm assuming the one that sees more forks will have a bigger Accounts size and thus run into the limit faster than the one that doesn't?

Reply:
The total is tracked in each bank, so whichever slots gets rooted is fine.

Reply:
Hmm so I'm guessing each bank only tracks the size of the account state on its particular fork, not the total account state across all forks on teh validator.

Reply:
💯% Correct!